packages/auth/src/accessToken/service.ts

```diff
@@ -65,3 +75,27 @@ async function introspect(
     return { active: true, ...grant, key: { proof: 'httpsig', jwk: keys[0] } }
   }
 }
+
+async function revoke(
+  deps: ServiceDependencies,
+  id: string
+): Promise<RevocationResult> {
```

Comment:
Why not return a boolean instead of the object?

Reply:
changed to return `Error | undefined`